tests/bluechi_test/test.py

```diff
@@ -82,14 +83,21 @@ def setup(self) -> Tuple[bool, Tuple[BluechiControllerContainer, Dict[str, Bluec
         return (success, (ctrl_container, node_container))
 
     def gather_logs(self, ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer]):
-        print("Collecting logs from all container...")
+        print("Collecting logs from all containers...")
 
         if ctrl is not None:
             ctrl.gather_journal_logs(self.tmt_test_data_dir)
 
         for _, node in nodes.items():
             node.gather_journal_logs(self.tmt_test_data_dir)
 
+        print("Collecting logs from test executor...")
+        self.gather_journal_logs()
+
+    def gather_journal_logs(self) -> None:
```

Comment:
```suggestion
    def gather_journal_logs_from_host(self) -> None:
```
I confused this with the `gather_journal_logs` on the container. Maybe we can rename it?

Reply:
OK, renamed